ert_gui/simulation/simulation_panel.py

```diff
@@ -68,6 +70,9 @@ def __init__(self, config_file):
 
         self.addSimulationConfigPanel(SingleTestRunPanel())
         self.addSimulationConfigPanel(EnsembleExperimentPanel())
+        if FeatureToggling.is_enabled("ensemble-evaluator"):
+            prefect_config = os.path.basename(config_file)[:-4] + ".yml"
+            self.addSimulationConfigPanel(PrefectEnsembleExperimentPanel(prefect_config))
```

Comment:
Maybe add support for the new config file in the command line parser? It is suggested by @sondreso that you then dump a dummy `.ert` file to disk to satisfy enkfmain.

Reply:
This tmp file probably needs to be created in the same folder as the other config, since ert will change folders if not.

Reply:
> Maybe add support for the new config file in the command line parser? It is suggested by @sondreso that you then dump a dummy .ert file to disk to satisfy enkfmain.

That would work :+1: